js/options.js

```diff
@@ -24,7 +24,16 @@ function save_options() {
 	var spamCount = document.getElementById('SpamCount').value;
 	var blockSpam = document.getElementById('BlockSpam').checked;
 	var omniSearch = document.getElementById('OmniSearch').checked;
+	//Quni Ticket Breakdown
 	var quniTickets = document.getElementById('QuniTickets').checked;
+	var siQueue = document.getElementById('SIQueue').checked;
+	var taQueue = document.getElementById('TAQueue').checked;
+	var three60Queue = document.getElementById('360Queue').checked;
+	var eeQueue = document.getElementById('EEQueue').checked;
+	var themesQueue = document.getElementById('ThemesQueue').checked;
+	var vocQueue = document.getElementById('VocQueue').checked;
+	var statQueue = document.getElementById('StatQueue').checked;
+	var intQueue = document.getElementById('IntQueue').checked;
```

Comment:
The variable names here don't really reflect what they are doing. Would be cleaner to encapsulate all of these in to a Queues object, then call a show or hide method on each when relevant. Like `Queues.taQueue.show`and `Queues.taQueue.hide`.